### ISSUE TITLE
Pretrained weights for other versions

### ISSUE THREAD

**Issue by JRestom** [sail-sg/poolformer#46]
Are there any available weights for the PoolFormer version with BN and LN instead of the Modified Layer Norm? (Table 5 in https://arxiv.org/abs/2111.11418)

**Comment by yuweihao** [comment]
Hi @JRestom ,

Thanks for your attention. For BN checkpoint, I have posted it in this issue #5 . For LN checkpoint, sorry that it is deleted. I will retrain it in a week and share it with you. Thanks.

**Comment by JRestom** [comment]
Thank you! It would be of great help.

**Comment by yuweihao** [comment]
Hi @JRestom ,

[poolformer_ln_s12](https://drive.google.com/file/d/1c0TIlfZgumXcKEQxNrnzpwIkLZ0UoH2P/view?usp=share_link) (let `norm_layer=LayerNormChannel`) is released. 

Thanks!

**Comment by JRestom** [comment]
Hello @yuweihao,
I highly appreciate the weights. One issue, when opening the link it directs me to a file called poolformer_bn_s12 (BatchNorm I assume). Could you double check the link?

**Comment by yuweihao** [comment]
Hi @JRestom , yes, [poolformer_bn_s12.pth.tar](https://drive.google.com/file/d/1c0TIlfZgumXcKEQxNrnzpwIkLZ0UoH2P/view) and let `norm_layer=torch.nn.BatchNorm2d`.

**Comment by JRestom** [comment]
I meant, the link of [poolformer_ln_s12.pth.tar](https://drive.google.com/file/d/1c0TIlfZgumXcKEQxNrnzpwIkLZ0UoH2P/view?usp=share_link) is directing me to the same folder as [poolformer_bn_s12.pth.tar](https://drive.google.com/file/d/1c0TIlfZgumXcKEQxNrnzpwIkLZ0UoH2P/view). Shouldn't they be different? (one for LayerNom and one for BatchNorm).

**Comment by yuweihao** [comment]
Oh, my fault, the right link is [poolformer_ln_s12.pth.tar](https://drive.google.com/file/d/1XWeScoZh8eOoWCg8qA1CvNj6SQyys7ou/view?usp=share_link).

**Comment by JRestom** [comment]
Thank you so much!!
